### ISSUE TITLE
onDragEnd not firing when moved to another dropZone

### ISSUE THREAD

**Issue by Mitchftw** [akserg/ng2-dnd#233]
* **I'm submitting a ...**
[x ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
bug

* **What is the current behavior?**
When an item from a sortable container is dropped somewhere outside of the div where it is not allowed to drop, onDragEnd is fired. However, this only happens when the object is still in it's original sortable container. When you move it to another dropZone and perform the same action, none of the eventEmitters kick in.

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via https://plnkr.co or similar.
described above

* **What is the expected behavior?**
onDragEnd should always fire

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**

- Angular version: 5.x
- Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by crooksey** [comment]
Possibly relates to issue #232 as this behaviour sounds identical, works for same sortable, but not moving to a new sortable.

**Comment by Mitchftw** [comment]
Indeed, it looks identical. I hope this will be fixed soon. Currently working on a workaround.

**Comment by nielszandsteeg** [comment]
Yes please, I have this issue too. The thing is that the arrays are already been sorted, but that doesnt mean the drop event will also be succesfull.

**Comment by nabc** [comment]
Agreed, onDragEnd should be fired whenever an item is dropped,sounds like issue #236

**Comment by crooksey** [comment]
I have added a temporary workaround/fix in #232
